kstreams-live-update/event-source/src/main/java/io/debezium/examples/kstreams/liveupdate/eventsource/EventSource.java

```diff
@@ -27,28 +25,34 @@ public void run() {
             EntityManager entityManager = entityManagerFactory.createEntityManager();
 
             entityManager.getTransaction().begin();
-            List<Station> stations = entityManager.createQuery("from Station s", Station.class).getResultList();
+            List<Category> categories = entityManager.createQuery("from Category c", Category.class).getResultList();
+            Object[] minMaxCustomerIds = (Object[]) entityManager.createNativeQuery("select min(id), max(id) from customers").getSingleResult();
+            Object[] minMaxProductIds = (Object[]) entityManager.createNativeQuery("select min(id), max(id) from products").getSingleResult();
+
             entityManager.getTransaction().commit();
 
             int i = 0;
             while (running) {
                 if (i % 50 == 0) {
-                    System.out.println("Inserted " + i + " measurements");
+                    entityManager.getTransaction().begin();
                 }
 
-                entityManager.getTransaction().begin();
-
-                entityManager.persist(getRandomMeasurement(entityManager, stations));
-                entityManager.getTransaction().commit();
+                entityManager.persist(getRandomOrder(entityManager, (int)minMaxCustomerIds[0], (int)minMaxCustomerIds[1], (int)minMaxProductIds[0], (int)minMaxProductIds[1], categories));
 
                 i++;
                 try {
-                    Thread.sleep(100);
+                    Thread.sleep(50);
                 }
                 catch (InterruptedException e) {
                     System.out.println("Interrupted");
                     running = false;
                 }
+
+                if (i % 50 == 0) {
+                    System.out.println("Inserted " + i + " measurements");
```

Comment:
Probably should not be *measurements*

Reply:
Done. Also changed it to use the logger.